### ISSUE TITLE
Address Vulnerabilities

### ISSUE THREAD

**Issue by JFania** [tishige/CallFlowVisualizer#5]
Hello,
I would like to try this app out but before doing so I had to have it reviewed by our software security team. They identified three high risk vulnerabilities. Would you be able to address these and complete the upgrades?

1. CVE-2024-30105 - upgrade to version System.Text.Json - 8.0.4
2. CVE-2024-21907 - upgrade to version Newtonsoft.json - 13.0.1
3. CVE-2024-27293 - upgrade to version RestSharp - 106.11.8-alpha.0.13

**Comment by tishige** [comment]
Hi,
In order to address this issue, I replaced these libraries with the latest versions.
However, due to changes in the specifications of Newtonsoft.json,
a certain flow causes an error and the process is interrupted, so I am using the old library.
I have inquired at the Genesys Developer Forum, but there has been no response so far.
https://developer.genesys.cloud/forum/t/need-to-change-newtonsoft-maxdepth-default-value/30495

Your flow may encounter errors, but I am planning to release a version with the latest libraries.
Once released, I will announce it on the following site:
https://community.genesys.com/discussion/creates-flowcharts-of-genesys-cloud-or-pureconnect-interaction-flow

Ishi
